packages/react-native-editor/__device-tests__/pages/editor-page.js

```diff
@@ -558,6 +558,29 @@ class EditorPage {
 		} while ( navigateUpElements.length > 0 );
 	}
 
+	// Adds a block by tapping on the appender button of blocks with inner blocks (e.g. Group block)
+	async addBlockUsingAppender( block, blockName ) {
```

Comment:
Can this new utility replace the existing Button-specific utility or do we need to retain both? 

https://github.com/WordPress/gutenberg/blob/91138d6b1e2dc9a4e98d1ccf4c56d0af48e9d499/packages/react-native-editor/__device-tests__/pages/editor-page.js#L991-L998

Reply:
Good question. I haven't tested this helper in the Buttons block, but I assume it will work because the appender buttons use the same test ID:

https://github.com/WordPress/gutenberg/blob/961d8ec6276019c05779600ce4b81e8bc8c2bf86/packages/block-library/src/buttons/edit.native.js#L118-L121

https://github.com/WordPress/gutenberg/blob/961d8ec6276019c05779600ce4b81e8bc8c2bf86/packages/block-editor/src/components/button-block-appender/index.native.js#L46

I'll keep this comment as a reminder to remove `addButtonWithInlineAppender` before setting the PR ready to review 👍 .

Reply:
I tried to replace `addButtonWithInlineAppender` with this function but didn't work as I expected. The inserter button of the Buttons block only requires tapping on it, while this functionality apart from tapping the button also selects a block fro the inserter menu.